cmd/cdi-importer/importer.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	contentType, _ := util.ParseEnvVar(common.ImporterContentType, false)
 	imageSize, _ := util.ParseEnvVar(common.ImporterImageSize, false)
 	certDir, _ := util.ParseEnvVar(common.ImporterCertDirVar, false)
+	insecureRegistry, _ := strconv.ParseBool(os.Getenv(common.InsecureRegistryVar))
```

Comment:
It doesn't appear that we're checking/enforcing that common.InsecureRegistryVar is a valid string representation of a bool, and we're ignoring the error; the result will be the default value for the assignment in the struct init below (so it'll be false), which is perfectly reasonable.

Just wonder if we should log something for users to know they screwed up?

Reply:
The value of this variable is computed by the import controller and `strconv.FormatBool` is used to set the value.  Not concerned about it containing a malformed value but I'll make sure that value is logged just in case.